### PR TITLE
fixing typos

### PR DESCRIPTION
**Pull request by dret**

### CODE REVIEW

File: webdriver-spec.html [hunk 1]
```diff
@@ -507,14 +507,14 @@
 <h2>Protocol</h2>
 
 <p>WebDriver <a>remote ends</a> must provide
- an HTTP-based, RESTish wire protocol
```
Comment (andreastt) [inline webdriver-spec.html]: Why remove?

File: webdriver-spec.html [hunk 2]
```diff
@@ -2326,7 +2326,7 @@
 <p class=note>The use of the term “window” to
  refer to a <a>top-level browsing context</a>
  is legacy and doesn’t correspond with either
- the operating system notaion of a “window”
+ the operating system notation of a “window”
```
Comment (andreastt) [inline webdriver-spec.html]: notion

**Comment by andreastt** [comment]
Please address the few comments I have, and I will merge this.

File: webdriver-spec.html (hunk 1 above)
Comment (dret) [inline webdriver-spec.html]: On 2016-04-27 02:41, Andreas Tolfsen wrote:
> In webdriver-spec.html
> <https://github.com/w3c/webdriver/pull/300#discussion_r61230067>:
>
>> @@ -507,14 +507,14 @@
>>  <h2>Protocol</h2>
>>
>>  <p>WebDriver <a>remote ends</a> must provide
>> - an HTTP-based, RESTish wire protocol
>
> Why remove?

why use it? it's not really REST, and what's REST or not is a confusing 
debate anyway. the term "RESTish" does not convey any information and at 
best it is harmless. or it does what it did to me, getting readers to 
think "now what's that supposed to mean?"

Comment (andreastt) [inline webdriver-spec.html]: OK, I’m happy with that explanation. Do you have any reservations, @AutomatedTester?

**Comment by dret** [comment]
On 2016-04-27 03:05, Andreas Tolfsen wrote:
> Please address the few comments I have, and I will merge this.

thanks, should be good to go now.

**Comment by dret** [comment]
thanks!

**Comment by AutomatedTester** [comment]
Thanks for the patch!

**Pull request merged**